velox/type/TimestampConversion.cpp

```diff
@@ -749,4 +750,13 @@ Timestamp fromTimestampString(const char* str, size_t len) {
   return timestamp;
 }
 
+std::tuple<const StringView&, std::string_view>
+splitTimestampWithTimezoneString(const std::string_view& tsTzStr) {
+  auto pos = tsTzStr.find(" ");
+  pos = tsTzStr.find(" ", pos + 1);
```

Comment:
Use `rfind`.

Reply:
Thanks, wasn't aware.

Reply:
Right. The latter would probably be a lot more feasible - I don't think it would be best to have `testCast()` use `evaluateOnce()` - as `evaluateCast()` is based on using the `CastTypedExpr` class which seems necessary due to the discrepancies in supported cast expressions between Velox and DuckDB.

I could open a separate issue to add a new `testCast()` signature to take in single values, which can be wrapped in vectors and passed on through the existing flow.